runtime/lang/menu_de_de.latin1.vim

```diff
@@ -1,312 +1,1267 @@
 
 " Quit when menu translations have already been done.
+
 if exists("did_menu_trans")
   finish
 endif
 let did_menu_trans = 1
-let s:keepcpo= &cpo
+let s:keepcpo = &cpo
 set cpo&vim
 
-" The translations below are in latin1, but they work for cp1252 and
-" iso-8859-15 without conversion as well.
-if &enc != "cp1252" && &enc != "iso-8859-15"
-  scriptencoding latin1
+scriptencoding latin1
+
+" Help menu
+menutrans &Help &Hilfe
+" Help menuitems and dialog {{{1
+menutrans &Overview<Tab><F1> &�berblick<Tab><F1>
+menutrans &User\ Manual &Handbuch
+menutrans &How-to\ Links How-to\ &Index
+menutrans &Find\.\.\. &Suchen\.\.\.	" conflicts with Edit.Find
+menutrans &Credits &Autoren
+menutrans Co&pying &Urheberrecht
+menutrans &Sponsor/Register Sponsern/&Registrieren
+menutrans O&rphans &Waisen
+menutrans &Version &Version
+menutrans &About &Titelseite
+
+" fun! s:Helpfind()
+if !exists("g:menutrans_help_dialog")
+  let g:menutrans_help_dialog = "Geben Sie einen Befehl oder ein Wort ein, f�r das Sie Hilfe ben�tigen:\n\nVerwenden Sie i_ f�r Eingabe ('input') Befehle (z.B.: i_CTRL-X)\nVerwenden Sie c_ f�r Befehls-Zeilen ('command-line') Befehle (z.B.: c_<Del>)\nVerwenden Sie ' f�r Options-Namen (z.B.: 'shiftwidth')"
 endif
+" }}}
 
-" {{{ FILE / DATEI
-menutrans &File				&Datei
-menutrans &Open\.\.\.<Tab>:e		&�ffnen\.\.\.<Tab>:e
-menutrans Sp&lit-Open\.\.\.<Tab>:sp	In\ geteiltem\ &Fenster\ �ffnen\.\.\.<Tab>:sp
-menutrans Open\ Tab\.\.\.<Tab>:tabnew	In\ neuem\ &Tab\ �ffnen\.\.\.<Tab>:tabnew
-menutrans &New<Tab>:enew		&Neue\ Datei<Tab>:enew
-menutrans &Close<Tab>:close		S&chlie�en<Tab>:close
-menutrans &Save<Tab>:w			&Speichern<Tab>:w
-menutrans Save\ &As\.\.\.<Tab>:sav	Speichern\ &als\.\.\.<Tab>:sav
-menutrans &Print			&Drucken
-menutrans Sa&ve-Exit<Tab>:wqa		Speichern\ und\ Be&enden<Tab>:wqa
-menutrans E&xit<Tab>:qa			&Beenden<Tab>:qa
-
-if has("diff")
-    menutrans Split\ &Diff\ with\.\.\.	D&ifferenz\ in\ geteiltem\ Fenster\ mit\.\.\.
-    menutrans Split\ Patched\ &By\.\.\.	&Patch\ in\ geteiltem\ Fenster\ mit\.\.\.
+" File menu
+menutrans &File &Datei
+" File menuitems {{{1
+menutrans &Open\.\.\.<Tab>:e &�ffnen\.\.\.<Tab>:e
+menutrans Sp&lit-Open\.\.\.<Tab>:sp In\ geteiltem\ &Fenster\ �ffnen\.\.\.<Tab>:sp
+menutrans Open\ Tab\.\.\.<Tab>:tabnew In\ neuem\ &Tab\ �ffnen\.\.\.<Tab>:tabnew
+menutrans &New<Tab>:enew &Neue\ Datei<Tab>:enew
+menutrans &Close<Tab>:close S&chlie�en<Tab>:close
+menutrans &Save<Tab>:w &Speichern<Tab>:w
+menutrans Save\ &As\.\.\.<Tab>:sav Speichern\ &als\.\.\.<Tab>:sav
+menutrans Split\ &Diff\ With\.\.\. D&ifferenz\ in\ geteiltem\ Fenster\ mit\.\.\.
+menutrans Split\ Patched\ &By\.\.\. &Patch\ in\ geteiltem\ Fenster\ mit\.\.\.
+menutrans &Print &Drucken
+menutrans Sa&ve-Exit<Tab>:wqa Speichern\ und\ Be&enden<Tab>:wqa
+menutrans E&xit<Tab>:qa &Beenden<Tab>:qa
+" }}}
+
+" Edit menu
+menutrans &Edit &Editieren
+" Edit menuitems {{{1
+menutrans &Undo<Tab>u Z&ur�ck<Tab>u
+menutrans &Redo<Tab>^R Vo&r<Tab>^R
```

Comment:
This should be `Wiederholen` or perhaps `Wiederherstellen`

Reply:
> This should be `Wiederholen` or perhaps `Wiederherstellen`

Thanks, ok, I will replace it.